ros_buildfarm/templates/snippet/publisher_warnings.xml.em

```diff
@@ -34,7 +33,8 @@
       <isEnabledForFailure>false</isEnabledForFailure>
       <isAggregatingResults>false</isAggregatingResults>
       <isBlameDisabled>false</isBlameDisabled>
-      <isForensicsDisabled>false</isForensicsDisabled>
+      <skipPublishingChecks>false</skipPublishingChecks>
```

Comment:
* warnings-ng has additional configuration for publishing checks to
GitHub. It is unclear if the successful use of this feature requires
usng the GitHub Branch Source plugin which we have not explored but the
if publishing checks works, we'll do it.